hexa/datasets/models.py

```diff
@@ -282,3 +289,11 @@ def delete_if_has_perm(self, *, principal: User):
 
     class Meta:
         unique_together = ("dataset", "workspace")
+
+
+class PipelineRunDatasetVersion(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    pipeline_run = models.ForeignKey("pipelines.PipelineRun", on_delete=models.CASCADE)
+    dataset_version = models.ForeignKey(
+        "datasets.DatasetVersion", on_delete=models.CASCADE
+    )
```

Comment:
You could add a ORM relationship to make it easier to query them

Reply:
Like ?